subsys/bluetooth/host/sdp.c

```diff
@@ -99,8 +110,10 @@ enum {
 
 struct search_state {
 	uint16_t att_list_size;
+	uint16_t last_att_offset;
 	uint8_t  current_svc;
 	uint8_t  last_att;
+	bool  rsp_buf_empty;
 	bool     pkt_full;
```

Comment:
It looks like the two bools aren't consistently aligned.

Reply:
Thanks. I will change it later.